src/test/java/de/mediathekview/mserver/crawler/hr/tasks/HrSendungsfolgedetailsTaskTest.java

```diff
@@ -0,0 +1,143 @@
+            },
+            {
+                "https://www.hessenschau.de/tv-sendung/hessenschau---ganze-sendung,video-73874.html",
+                "/hr/hr_film_detail_hessenschau.html",
+                "hessenschau",
+                "hessenschau - ganze Sendung",
+                "- Gründung der \"Juden in der AfD\" von Protesten begleitet - Landesgartenschau schließt ihre Pforten - Vereine verwandeln Altpapier in Bares - Weitergedreht: Räumungs-Wirrwarr in Offenbach - Modemacher (7): Handschuh-Designerin aus Frankfurt",
+                LocalDateTime.of(2018, 10, 7, 19, 30, 0),
+                Duration.ofMinutes(27).plusSeconds(30),
+                "https://hr-a.akamaihd.net/video/as/hessenschau/2018_10/181007203848_2018-10-07_19-25-02_hessenschau_512x288-25p-500kbit.mp4",
+                "https://hr-a.akamaihd.net/video/as/hessenschau/2018_10/181007203848_2018-10-07_19-25-02_hessenschau_960x540-50p-1800kbit.mp4",
+                "https://hr-a.akamaihd.net/video/as/hessenschau/2018_10/181007203848_2018-10-07_19-25-02_hessenschau_1280x720-50p-5000kbit.mp4",
+                "https://hr-a.akamaihd.net/video/as/hessenschau/2018_10/181007203848_2018-10-07_19-25-02_hessenschau_512x288-25p-500kbit.vtt",
+                GeoLocations.GEO_NONE
+            },
+            {
+                "https://www.hr-fernsehen.de/sendungen-a-z/hessenreporter/sendungen/im-einsatz-gegen-mietwucher-die-sozialfahnderin,sendung-34026.html",
+                "/hr/hr_film_detail3.html",
+                "Hessenreporter",
+                "Im Einsatz gegen Mietwucher",
+                "Drei Familien mit insgesamt elf Personen in einer Fünf-Zimmer-Wohnung - ohne Heizung, ohne Küche, die einzige Toilette auf halber Treppe im Hausflur: Solches Elend sieht Petra Windrich fast täglich. Der Vermieter kassiert von jeder Partei 700 Euro, insgesamt also 2.100 Euro – jeden Monat. Ein Fall von Mietwucher.",
+                LocalDateTime.of(2018, 5, 28, 18, 30, 0),
+                Duration.ofMinutes(29).plusSeconds(30),
+                "",
+                "https://hr-a.akamaihd.net/video/as/hessenreporter/2018_05/hrLogo_180528130435_0197529_512x288-25p-500kbit.mp4",
+                "",
+                "https://hr-a.akamaihd.net/video/as/hessenreporter/2018_05/hrLogo_180528130435_0197529_512x288-25p-500kbit.vtt",
+                GeoLocations.GEO_NONE
+            }
+        });
+  }
+
+  private final String requestUrl;
+  private final String htmlPage;
+  private final String expectedTopic;
+  private final String expectedTitle;
+  private final String expectedDescription;
+  private final LocalDateTime expectedTime;
+  private final Duration expectedDuration;
+  private final String expectedUrlSmall;
+  private final String expectedUrlNormal;
+  private final String expectedUrlHd;
+  private final String expectedSubtitle;
+  private final GeoLocations expectedGeo;
+
+  public HrSendungsfolgedetailsTaskTest(final String aRequestUrl, final String aHtmlPage,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 12 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)